### PR TITLE
Migrated to AndroidX

### PR DESCRIPTION
**Pull request by MonikaJethani**
Migrated project containing library and demo modules to AndroidX.

### CODE REVIEW

**Comment by MonikaJethani** [comment]
@sakulisko Please review this PR.

File: build.gradle [hunk 1]
```diff
@@ -1,16 +1,24 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
```
Comment (sakulisko) [inline build.gradle]: you can use google()

File: library/src/main/java/com/avast/android/dialogs/core/BaseDialogFragment.java [hunk 2]
```diff
@@ -26,12 +26,14 @@
 import android.content.res.TypedArray;
 import android.graphics.Typeface;
 import android.os.Bundle;
-import android.support.annotation.NonNull;
-import android.support.annotation.StyleRes;
-import android.support.v4.app.DialogFragment;
-import android.support.v4.app.Fragment;
-import android.support.v4.app.FragmentManager;
-import android.support.v4.app.FragmentTransaction;
+
```
Comment (sakulisko) [inline library/src/main/java/com/avast/android/dialogs/core/BaseDialogFragment.java]: Please keep this order of imports 
- java*
- empty line
- android.*
- empty line
- androidx.*
- empty line
- other

**Comment by sakulisko** [comment]
Hi, thanks for PR.
To pass CI please try to update `.travis.yml` with this content
```
language: android
script: "./gradlew assembleDebug"
android:
  components:
    - build-tools-28.0.3
    - android-28
    - extra-android-m2repository
```

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.2.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
```
Comment (sakulisko) [inline gradle/wrapper/gradle-wrapper.properties]: please run ./gradlew :wrapper to update gradle wrapper scripts and jar

File: build.gradle (hunk 1 above)
Comment (MonikaJethani) [reply build.gradle]: Done.

File: library/src/main/java/com/avast/android/dialogs/core/BaseDialogFragment.java (hunk 2 above)
Comment (MonikaJethani) [reply library/src/main/java/com/avast/android/dialogs/core/BaseDialogFragment.java]: Done.

File: gradle/wrapper/gradle-wrapper.properties (hunk 3 above)
Comment (MonikaJethani) [reply gradle/wrapper/gradle-wrapper.properties]: Done.

**Comment by MonikaJethani** [comment]
> Hi, thanks for PR.
> To pass CI please try to update `.travis.yml` with this content
> 
> ```
> language: android
> script: "./gradlew assembleDebug"
> android:
>   components:
>     - build-tools-28.0.3
>     - android-28
>     - extra-android-m2repository
> ```

Hi @sakulisko. travis file is updated and checks are passed. All other review comments have been implemented. Please do a final review and merge. Thanks.

**Comment by sakulisko** [comment]
@MonikaJethani Thank you for you contribution.

**Pull request merged**